Sources/TuistKit/Commands/GraphCommand.swift

```diff
@@ -102,6 +106,18 @@ enum GraphFormat: String, ExpressibleByArgument, CaseIterable {
     case dot, json, legacyJSON, png, svg
 }
 
+extension GraphFormat {
```

Comment:
Nit: Since this is only used by the service, I'd move it there and make it private.

Reply:
Moved it and made it private